monicar-event-hub/src/main/java/org/eventhub/presentation/OnOffController.java

```diff
@@ -96,6 +98,12 @@ public BaseResponse keyOff(
 			request.offTime()
 		));
 
+		Optional<Long> alarmId = alarmService.saveAlarmIfNecessary(
+			vehicleInformation.getId(),
+			updatedTotalDistance
+		);
+		alarmId.ifPresent(alarmService::sendAlarm);
```

Comment:
```suggestion
		alarmService.saveAlarmIfNecessary(vehicleInformation.getId(), updatedTotalDistance)
		    .ifPresent(alarmService::sendAlarm);
```
한 줄로 하면 별도로 변수 선언 없이 가능할 것 같네용

Reply:
좋은데요?!! 반영했십니다